packages/e2e-tests/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Container package for end to end tests, from contracts to browser apps",
   "main": "index.js",
   "scripts": {
-    "test": "wait-on http://localhost:3000 http://localhost:3055 && jest",
+    "test": "wait-on http://localhost:3000 http://localhost:3055 && jest --runInBand",
```

Comment:
Why's this?

Reply:
Because the individual tests themselves could conflict (there are 3) if run in parallel, I think